### PR TITLE
#287 Range for NaturalNumbers

### PR DESCRIPTION
**Pull request by fabriciofx**
As per #287.

### CODE REVIEW

File: src/test/java/org/cactoos/list/NaturalNumbersTest.java [hunk 1]
```diff
@@ -56,4 +56,25 @@ public void notStartsWithNegativeNumbers() {
             new ScalarHasValue<>(0L)
         );
     }
+
+    @Test
+    public void containsNaturalNumbersRange() {
+        MatcherAssert.assertThat(
+            "Can't get range of natural numbers",
+            // @checkstyle MagicNumber (2 lines)
```
Comment (VsSekorin) [inline src/test/java/org/cactoos/list/NaturalNumbersTest.java]: This string is repeated three times in this class. Perhaps this should be moved to class javadoc? Like

> checkstyle JavadocMethodCheck (500 lines)

File: src/main/java/org/cactoos/list/NaturalNumbers.java [hunk 2]
```diff
@@ -35,8 +35,43 @@
  */
 public final class NaturalNumbers implements Iterable<Long> {
 
+    /**
+     * First natural number.
+     */
+    private final long first;
```
Comment (fabriciofx) [inline src/main/java/org/cactoos/list/NaturalNumbers.java]: @VsSekorin Ok. I'll change it. Thanks!

File: src/test/java/org/cactoos/list/NaturalNumbersTest.java (hunk 1 above)
Comment (fabriciofx) [inline src/test/java/org/cactoos/list/NaturalNumbersTest.java]: @VsSekorin I don't think so. Will it aggregate value to this software? WDYT?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**